package.json

```diff
@@ -33,6 +33,8 @@
     "should": "3.3.2",
     "xunit-file": "0.0.5"
   },
+  "preferGlobal": true,
```

Comment:
I think having the user explicitly pass in `-g` is a better option, which seems to be what you documented in the README.